$OpenBSD: patch-Modules_FindOpenMP_cmake,v 1.3 2020/05/25 05:12:00 rsadowski Exp $

Add libs and paths. Ensure the lib is used for the C case.

Index: Modules/FindOpenMP.cmake
--- Modules/FindOpenMP.cmake.orig
+++ Modules/FindOpenMP.cmake
@@ -95,8 +95,8 @@ function(_OPENMP_FLAG_CANDIDATES LANG)
   if(NOT OpenMP_${LANG}_FLAG)
     unset(OpenMP_FLAG_CANDIDATES)
 
-    set(OMP_FLAG_GNU "-fopenmp")
-    set(OMP_FLAG_Clang "-fopenmp=libomp" "-fopenmp=libiomp5" "-fopenmp" "-Xclang -fopenmp")
+    set(OMP_FLAG_GNU "-fopenmp -lgomp")
+    set(OMP_FLAG_Clang "-fopenmp -I${LOCALBASE}/include -Wl,-L${LOCALBASE}/lib,-lomp")
     set(OMP_FLAG_AppleClang "-Xclang -fopenmp")
     set(OMP_FLAG_HP "+Oopenmp")
     if(WIN32)
